x-pack/docs/en/security/securing-communications/security-basic-setup.asciidoc

```diff
@@ -0,0 +1,277 @@
+
+   b. Enter a password for your CA.
+
+. In the installation directory on one your {es} nodes, create a
+`certificates.yaml` file to contain information about each node.
++
+Enter a `name` value for each node in your cluster, which can be a simple string value or a Distinguished Name (DN). This is the only required field.
++
+[source,yaml]
+----
+instances:
+ - name: "node-1"
+ - name: "node-2"
+ - name: "node-3"
+----
+
+. Run the `elasticsearch-certutil` tool in `--silent` mode with the `cert`
+parameter and the following options:
++
+[source,shell]
+----
+./bin/elasticsearch-certutil cert --ca elastic-certificates.p12 --silent --in certificates.yaml --out node-certs.zip --pass <password>
+----
++
+This command generates a compressed `node-certs.zip` file.
+
+   `--ca <ca_file>`:: Name of the CA file used to sign your certificates. The
+   default file name from the `elasticsearch-certutil` tool is `elastic-certificates.p12`.
+
+   `--in <input_file>`:: Name of the file that is used to run in https://www.elastic.co/guide/en/elasticsearch/reference/current/certutil.html#certutil-silent[silent mode]. The input file must be a YAML file. This parameter cannot be used with the `ca` parameter.
+
+   `--out <file_path>`:: Path and file name for the output files.
+
+   `--pass <password>`:: Password for the generated private keys.
+
+. Enter the password for your CA when prompted.
+
+. Unzip the `node-certs.zip` output file, which results in one directory for each node that was included in the `certificates.yaml` file.
++
+Each node directory contains a single `<node-name>.p12` certificate file, which contains the node certificate, instance private key, and CA certificate.
+
+. Copy the appropriate `<node-name>.p12` certificate file to the
+   `ES_PATH_CONF` directory on each node.
++
+WARNING: This file must reside in the `ES_PATH_CONF` directory or {es}
+will not start.
+
+*Next*: <<encrypt-internode-communication>>
+
+[[encrypt-internode-communication]]
+==== Encrypt internode communications with TLS
+
+The transport networking layer is used for internal communication between
+nodes in a cluster. When security features are enabled, you must use TLS to
+ensure that communication between the nodes is encrypted.
+
+Now that you've generated a certificate authority and certificates, you'll
+update your cluster to use these files.
+
+NOTE: Complete the following steps for each node in your cluster. To join the
+same cluster, all nodes must share the same `cluster.name` value.
+
+.  Open the `ES_PATH_CONF/elasticsearch.yml` file and make the following
+changes:
+
+   a. Add the `cluster-name` setting and enter a name for your cluster:
++
+[source,yaml]
+----
+cluster.name: my-cluster
+----
+
+b. Add the `node.name` setting and enter the name of the certificate that
+you generated for this node. The value must match the certificate name that
```

Comment:
Agreed, the way it is written here it implies that we do some kind of check in TLS that means that the certificate with the name X can be used only in the node with name X , which makes sense but is not necessary

Reply:
I think that we can suggest that users have the same value for the certificate name, rather than say it must match. Something like:
>For simplicity, it's good practice for this value to match the certificate name that you defined in your `certificates.yaml` file.